frontend/public/locales/en/translation.json

```diff
@@ -3,5 +3,16 @@
     "description": {
         "part1": "To get started, edit <1>src/App.js</1> and save to reload.",
         "part2": "Switch language between english and german using buttons above."
+    },
+    "translation": {
+        "page_names": {
+            "projects": "Projects",
+            "leaders": "Leaders",
+            "statistics": "Statistics",
+            "settings": "Settings",
+            "users": "User's Settings",
+            "tags": "Tags",
+            "handbook": "Handbook"
```

Comment:
I'll discuss term "handbook" with Andrei. It seems like there should be a better word for this concept... But now we have no better option...